component-tools/src/main/java/org/talend/sdk/component/tools/validator/ModelValidator.java

```diff
@@ -102,15 +100,10 @@ private int countParameters(final Parameter[] params) {
         return (int) Stream.of(params).filter((Parameter p) -> !this.helper.isService(p)).count();
     }
 
-    private boolean isMapString(final Field f) {
-        final ParameterizedType pt = ParameterizedType.class.cast(f.getGenericType());
-        return (Map.class == pt.getRawType()) && pt.getActualTypeArguments().length == 2
-                && pt.getActualTypeArguments()[0] == String.class && pt.getActualTypeArguments()[1] == String.class;
-    }
-
-    private boolean isListString(final Field f) {
+    private boolean isListObject(final Field f) {
         final ParameterizedType pt = ParameterizedType.class.cast(f.getGenericType());
         return ((List.class == pt.getRawType()) || (Collection.class == pt.getRawType()))
-                && pt.getActualTypeArguments().length == 1 && pt.getActualTypeArguments()[0] == String.class;
+                && (pt.getActualTypeArguments().length == 1) && (pt.getActualTypeArguments()[0] instanceof Object);
```

Comment:
A Collection (or List) will always have arguments type that are object instance (List is transform to List<Object>)
Why not accept "pt.getRawType() as a subtype of Iterable" (Iterable.isAssignableFrom(pt.getRawType())

instanceof with a type ? more Object.class.isAssignableFrom(xxx) => both case will always true